lib/bloc/user_profile/user_profile_bloc.dart

```diff
@@ -160,6 +162,23 @@ class UserProfileBloc {
     return user;
   }
 
+  _updateBiometricsSettings() {
+    _checkBiometrics();
+    _deviceService.eventStream.listen((e){
+      if (e == NotificationType.RESUME) {
+        _checkBiometrics();
+      }
+    });
+  }
+
+  Future _checkBiometrics() async {
+    String enrolledBiometrics = await _localAuthService.enrolledBiometrics;
+    if (enrolledBiometrics == "" || enrolledBiometrics != _currentUser.securityModel.enrolledBiometrics) {
+      _updateSecurityModel(UpdateSecurityModel(_currentUser.securityModel.copyWith(isFingerprintEnabled: false)));
+    }
+    _updateSecurityModel(UpdateSecurityModel(_currentUser.securityModel.copyWith(enrolledBiometrics: enrolledBiometrics)));
```

Comment:
We don't need to take care of biometric settings changes.
Maybe this is better?
```
bool isFingerprintEnabled = _currentUser.securityModel.isFingerprintEnabled && enrolledBiometrics.isNotEmpty
_updateSecurityModel(UpdateSecurityModel(_currentUser.securityModel.copyWith(enrolledBiometrics: enrolledBiometrics, isFingerprintEnabled: isFingerprintEnabled)));
```

Reply:
[Done](https://github.com/breez/breezmobile/pull/210/commits/7efadec40ee2b25b11fb4791048cee014edf998c)